.gitignore

```diff
@@ -21,5 +21,4 @@ OMPublicKey/
 OMPython/
 OMSetup/
 OpenModelicaSetup/
-<<<<<<< HEAD
-OMEncryption/
```

Comment:
Remove this git merge conflict marker.

Reply:
That *was* removed, it was there (as you can see these are deletions, not additions).

Reply:
I will however add back OMEncryption to the ignore list.

Reply:
Nevermind, OMEncryption was already there, a bit further up.